### PR TITLE
feat: aries-js-worker accepts and returns objects

### PR DESCRIPTION
**Pull request by llorllale**
* accept and return objects
  * if arg is a string, parse it as JSON

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/main.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ import (
 	"syscall/js"
 	"time"
 
+	"github.com/mitchellh/mapstructure"
+
```
Comment (troyronda) [inline cmd/aries-js-worker/main.go]: extra space

File: cmd/aries-js-worker/index.html [hunk 2]
```diff
@@ -98,15 +98,15 @@
                     <label for="operation">Operation: </label>
                 </td>
                 <td>
-                    <input type="text" id="operation" value="didexchange" size="75"></input>
+                    <input type="text" id="operation" value="_test" size="75"></input>
```
Comment (troyronda) [inline cmd/aries-js-worker/index.html]: did you intend to leave this change?

File: cmd/aries-js-worker/main.go [hunk 3]
```diff
@@ -94,7 +98,7 @@ func takeFrom(in chan *command) func(js.Value, []js.Value) interface{} {
 }
 
 func pipe(input chan *command, output chan *result) {
-	handlers := defaultHandlers()
+	handlers := testHandlers()
```
Comment (troyronda) [inline cmd/aries-js-worker/main.go]: testHandlers?

**Review by troyronda — APPROVED** [review]

File: cmd/aries-js-worker/main.go (hunk 3 above)
Comment (sudeshrshetty) [reply cmd/aries-js-worker/main.go]: in my previous PR @llorllale asked to leave some test handlers behind to test JS wasm integration. May be time to take them off now?

**Review by sudeshrshetty — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/aries-js-worker/index.html (hunk 2 above)
Comment (llorllale) [reply cmd/aries-js-worker/index.html]: Yup

File: cmd/aries-js-worker/main.go (hunk 3 above)
Comment (llorllale) [reply cmd/aries-js-worker/main.go]: No, not yet

File: cmd/aries-js-worker/main.go (hunk 1 above)
Comment (llorllale) [reply cmd/aries-js-worker/main.go]: The linters should be fixed

File: cmd/aries-js-worker/index.html [hunk 4]
```diff
@@ -31,7 +31,7 @@
             const operation = document.querySelector("#operation").value
             const method = document.querySelector("#method").value
             const response = await aries[operation][method](request)
-            document.querySelector("#output").value = response
+            document.querySelector("#output").value = JSON.stringify(response)
```
Comment (troyronda) [inline cmd/aries-js-worker/index.html]: I wonder if it's possible to hide stringify in the library. https://github.com/hyperledger/aries-framework-go/blob/master/cmd/aries-js-worker/src/aries.js#L60

**Pull request merged**